### ISSUE TITLE
FR: Rename Developer Tools to System Tools

### ISSUE THREAD

**Issue by Tediore** [home-assistant/frontend#6316]
## The request

A few of us on Discord have been discussing that Developer Tools is a misleading term (since you don't actually need to be a developer to use those tools or to use Home Assistant in general). Renaming that page to System Tools or something similar would better describe what those tools are and would also be a friendlier name.

## The alternatives

N/A
## Additional information
N/A

**Comment by Tediore** [comment]
Something else I thought of... if we do rename it to System Tools, I think it would be logical to move Logs and Info back to that section as well. Configuration seems like an odd place for those items (since there's nothing to actually configure in the Logs and Info panels). I realize this would basically undo most of what was changed with Developer Tools in 0.112, but I figured it can't hurt to suggest.

**Comment by m3m0rex** [comment]
pls also fix the directions:

Config error. See the logs for details. 

current : /developer-tools/logs
right value: /config/logs

**Comment by SeanPM5** [comment]
I agree with this, but kind of think that "System" might still be a little scary for some users? When I help friends and family with computer stuff, they often equate the word system to "nerd stuff" heh.

I lean towards just naming it "Tools" as that would make it more approachable to newcomers, probably easier to translate, and would also future proof it somewhat (because in the future we could add more tools that aren't necessarily system-related). 

But I'm fine with either.

**Comment by oncleben31** [comment]
What about `Advanced tools` ? And perhaps visibility of this tools could be linked to `Advanced mode` in user settings.

**Comment by Villhellm** [comment]
The point is that these are not really “advanced” or “developer” specific features. They are a necessity to anyone that wants to start automating. Which is the entire point of Home Assistant to begin with.

I’m on board with “Tools” or “Toolbox”. Should be nonthreatening to new users as much as possible.

**Comment by Tediore** [comment]
I don't think System Tools would be too intimidating, but I can understand how it might appear that way to some people. However, to your suggestion @SeanPM5 about simply renaming it to "Tools," Villhellm brought up a good point in private discussion that making it one word would align it with the other items in the sidebar (Overview, Logbook, History, etc). For some reason making it one word feels odd to me, but the logic behind it makes sense.

**Comment by bramkragten** [comment]
@Villhellm I don't agree the developer tools are a necessity to use Home Assistant or start automating, and I do think it is very advanced.

We are trying really hard to make things easier with device automations and other tools in the UI so you don't need these tools.

**Comment by allan-reynolds** [comment]
To bring it inline with many other types of system paradigm, would it not be better to rename "Configuration" to "Settings"  and move the "Developer tools" into a section at the bottom of that page?  

This would have the added bonus of freeing space on the left hand bar ( as I am sure many others are like mine, quite full these days)

**Comment by zsarnett** [comment]
I like this idea. Settings and then an Advanced Settings inside of that. This is similar to how the android settings are

**Comment by Tediore** [comment]
Not sure I agree with that, the developer tools aren't settings.

**Comment by zsarnett** [comment]
Not all of the Developer options in the Android settings are either. Its more this is where everything is going to be. One place all settings, tools,  automations

**Comment by Tediore** [comment]
But they're part of your configuration, hence that section being named Configuration. This discussion is probably out of scope for this FR question though.

**Comment by zsarnett** [comment]
I don't think this is out of scope. If we are to change these let's change it once instead of changing the name and then changing the placement (if we do). Even if we do them in separate releases its better to get the plan for both done first.

Here is my summary:

I like Settings. I think it normalizes it and makes it easier to know what you are clicking but I am also not against leaving as Configuration

I think that Advanced Tools should go in settings. While these aren't Settings it's where I would expect anything advanced to live. Not something that is usually a click away. Advanced anything is usually behind 1 if not 2 button clicks.

By putting Advanced Tools in the settings this frees up space in the UI navigation. And this is where I think we should have an `Automation` menu. Here is where I think Automations, Scripts, Scenes, etc. As these are all part of automation within home assistant.

As Home Assistant was built for automation I think that this menu is much more important than advanced tools and should be better displayed / easier to get to

**Comment by Villhellm** [comment]
Even with device automations there are tools in what is currently called Developer Tools that are necessary to do almost any amount of debugging and testing for automations ie checking if a service is working, or making sure an entity is actually changing state properly. Hiding them deeper in menus is just going to frustrate the people that are trying to fix/test their automations and will be limiting them to a very small subset of capability. Saying that they "aren't necessary" is a naive outlook on how these tools are/should used, in my opinion.

There are currently ways to hide, rename, rearrange, and add items to the sidebar if anyone is really worried about the real estate there. In the end, it doesn't *really* matter. But suggesting a complete rework of the configuration menus is absolutely out of scope for a FR that was suggesting a simple name change.

**Comment by zsarnett** [comment]
With that, if there is anyone who wants to test and check the automation etc then they can always add the item to their sidebar. As Bram said the goal is to not need those tools. While it may be out of scope until we get there. I still don't think these tools are very necessary to the avg user. I wrote all of my automation without them and the only reason I have ever used them is for developing.

Feature Requests are to spark conversation. The FR is not always what the end result should be. This FR really opened up the conversation for these tools and what they are.

and like I said changing them and then changing them again is more frustrating then changing them once. Getting a plan for how the menus should be is beneficial here.

**Comment by matthiasdebaat** [comment]
> I agree with this, but kind of think that "System" might still be a little scary for some users? When I help friends and family with computer stuff, they often equate the word system to "nerd stuff" heh.
> 
> I lean towards just naming it "Tools" as that would make it more approachable to newcomers, probably easier to translate, and would also future proof it somewhat (because in the future we could add more tools that aren't necessarily system-related).

I like `Tools`. it just describes what it is and is neutral for who wants to use it.

**Comment by VNRARA** [comment]
Why not just calling it Testing Tools or Service Tools
